bevm/bevmclient/README.md

```diff
@@ -0,0 +1,40 @@
+Navigation: [DEDIS](https://github.com/dedis/doc/tree/master/README.md) ::
+[Cothority](../README.md) ::
+[Building Blocks](../doc/BuildingBlocks.md) ::
+[BEvm](https://github.com/dedis/cothority/blob/master/bevm/README.md) ::
+bevmclient
+
+# bevmclient - CLI tool to deploy and interact with BEvm contracts
+
+For the details on all the options and arguments, invoke the tool using the `--help` option.
+
+## Create a BEvm account
+Assuming ByzCoin config and key files in the current directory (see [bcadmin](https://github.com/dedis/cothority/blob/master/byzcoin/bcadmin/README.md) for details):
+```bash
+bevmclient create_account --account-name <MyAccount>
+```
+
+## Credit a BEvm account
+```bash
+bevmclient --config . credit_account --bc bc-<ByzCoinID>.cfg --bevm-id <BEvm instance ID> --account-name <MyAccount> <amount>
+```
+
+## Retrieve the balance of a BEvm account
+```bash
+bevmclient --config . get_account_balance --bc bc-<ByzCoinID>.cfg --bevm-id <BEvm instance ID> --account-name <MyAccount>
+```
+
+## Deploy a BEvm contract
+```bash
+bevmclient --config . deploy_contract --bc bc-<ByzCoinID>.cfg --bevm-id <BEvm instance ID> --account-name <MyAccount> --conjtract-name <MyContract> <ABI file> <bytecode file> [<arg>...]
+```
+
+## Execute a transaction on a deployed BEvm contract instance
+```bash
+bevmclient --config . transaction --bc bc-<ByzCoinID>.cfg --bevm-id <BEvm instance ID> --account-name <MyAccount> --contract-name <MyContract> <transaction name> [<arg>...]
```

Comment:
I don't know if this is a valid concern but it feels like this command is pretty lengthy with all the flags and arguments. Does it make sense to use a .toml file as we do for [running simulations on Onet? (https://github.com/dedis/onet/tree/master/simul#runfile-for-simulations)]

Reply:
Hmm - in the case of simulations, the same `.toml` file is used again and again. While here, you will not send always the same transaction (except for simulations ;). So I think it's OK to have this longish command...

One way would be to put more information in the configuration itself, as probably the `bc` and `bevm-id`will not change a lot but usually stay the same.

Reply:
@ineiti good point - you're right about it. and yes as you said, it should be possible to at least get rid of `bc` and `bevm-id`.

Reply:
Good point. I have also been pondering about this...
Both `--bc` and `--bevm-id` support an environment variable (as illustrated by usage in `test.sh`).
I did not add this to the config file, not wanting to "pollute" the config file with BEvm stuff, but if you think that's OK I can do that.

Reply:
I would not add that kind of specific stuff to the bc file, which is widely used for other tasks not concerned by BEvm. Using environment variables should do the job.